core/pom.xml

```diff
@@ -33,7 +33,7 @@
     <url>https://arctic.netease.com</url>
 
     <properties>
-        <rocksdb.version>5.17.2</rocksdb.version>
+        <rocksdb.version>7.10.2</rocksdb.version>
```

Comment:
I test it locally on M1 chip(arm64), and it works. 👍

Reply:
Why should we change the rocksdb version?
PR #1558  changes the version to `6.20.3-ververica-1.0`, I am not sure dose this will affect this PR.

Reply:
Getting rocksdb to work on M1 chip, which I mentioned in #1194